app/services/process_feed_service.rb

```diff
@@ -132,6 +132,8 @@ def status_from_xml(entry)
 
       return [nil, false] if account.suspended?
 
+      conversation = find_or_create_conversation(entry)
```

Comment:
It looks like this local variable is only ever used once as an arg to `conversation: ...` ... maybe we don't need the local var, and the assignment could just be from the method?